src/main/java/micdoodle8/mods/galacticraft/core/entities/player/FreefallHandler.java

```diff
@@ -420,7 +420,7 @@ public void postVanillaMotion(EntityPlayerSP p)
         }
         boolean doCentrifugal = spinManager != null;
 
-        if (freefall)
+        /*f (freefall)
```

Comment:
Please don't just comment out code, if it really should be removed, then remove the code itself.